spec/models/queue_item_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'spec_helper'
+
+describe QueueItem do 
+  it { should belong_to(:user)}
+  it { should belong_to(:video)}
+
+  describe '#video_title' do 
+    it 'return the title of the accossiated video' do 
+      video = Fabricate(:video, title: 'Monk')
+      queue_item = Fabricate(:queue_item, video: video)
+      expect(queue_item.video_title).to eq('Monk')
+    end
+  end
+
+  describe '#rating' do 
+    it 'returns the rating from the review when the review is present' do 
+      video = Fabricate(:video)
+      user =Fabricate(:user)
+      review = Fabricate(:review, video: video, user: user)
+      queue_item = Fabricate(:queue_item, video: video, user: user)
+      expect(queue_item.rating).to eq(review.rating)
+    end
```

Comment:
I would recommend having 1 empty line between examples. It feels like you give it more room and its more readable.